docs/containers.conf.5.md

```diff
@@ -50,6 +50,12 @@ Files in those locations are not loaded by default but only on-demand.  They are
 
 Modules are currently supported by podman(1).  The `podman --module` flag allows for loading a module and can be specified multiple times.  If the specified value is an absolute path, the config file will be loaded directly.  Relative paths are resolved relative to the three module directories mentioned above and in the specified order such that modules in `$HOME` allow for overriding those in `/etc` and `/usr/share`.  Modules in `$HOME` (or `$XDG_CONFIG_HOME` if specified) are only used for rootless users.
 
+## APPENDING TO STRING ARRAYS
+
+The default behavior during the loading sequence of multiple containers.conf files is to override previous data.  Assume configuration 1 sets `env=["1=true"]` and configuration 2 sets `env=["2=true"]`.  The final setting will contain the data of configuration 2 as it overrides the previous settings.
+
+To change the behavior from overriding to appending, you can set the `append` attribute as follows: `array=["item-1", "item=2", ..., {append=true}]`.  Setting the append attribute instructs to append to this specific string array for this and also subsequent loading steps.  Hence, setting it to configuration 1 or configuration 2 will have the same effect and lead to `env=["1=true", "2=true"]`.  To change back to overriding, set `{append=true}`. 
```

Comment:
```suggestion
To change the behavior from overriding to appending, you can set the `append` attribute as follows: `array=["item-1", "item=2", ..., {append=true}]`.  Setting the append attribute instructs to append to this specific string array for this and also subsequent loading steps.  Hence, setting it to configuration 1 or configuration 2 will have the same effect and lead to `env=["1=true", "2=true"]`.  To change back to overriding, set `{append=false}`. 
```

While this sentence is right I think it may lead users to a wrong impression that it does not matter where to set the append attribute. Assuming we have 3 config files and the user sets append only in the last one then they end up with values from 2 and 3 but NOT 1. I think that should be clarified here.

Reply:
Yes I think I would need an example of multiple config files.

grep env modules* .conf
modules1.conf: env=["1=true",]
modules2.conf: env=["2=true", {append=true},]
modules3.conf: env=["3=true",]
modules3.conf: env=["4=true",]

What does 
podman --modules=modules1 --modules=modules2 --modules=modules3 --modules=modules4 run fedora printenv `

Produce

Reply:
Better?